chia/types/blockchain_format/program.py

```diff
@@ -32,11 +33,18 @@ def stream(self, f):
         sexp_to_stream(self, f)
 
     @classmethod
-    def from_bytes(cls, blob: bytes) -> "Program":
-        f = io.BytesIO(blob)
-        result = cls.parse(f)  # noqa
-        assert f.read() == b""
-        return result
+    def from_bytes(cls, blob: bytes) -> Program:
```

Comment:
```suggestion
    def from_bytes(cls, blob: bytes) -> "Program":
```

I thought we had to quote `Program` here because it's not in scope until the entire `Program` class is parsed.

Reply:
that's true, unless you `from __future__ import annotations` at the top